tl/appendices/3-0-migration-guide.rst

```diff
@@ -476,178 +476,198 @@ Tugon
-  public methods can be used as controller actions.
-
-Scaffold Removed
-----------------
-
-The dynamic scaffolding in CakePHP has been removed from CakePHP core. It was
-infrequently used, and never intended for production use. An improved
-scaffolding plugin, named CRUD, can be found here:
+- Ang ``$helpers``, ``$components`` na mga katangian ay na-merge na ngayon
+  kasama ang **lahat** ng magulang na mga class hindi lang ang ``AppController``
+  at ang plugin na AppController. Ang mga katangian din ay magkaibang na-merge.
+  Sa halip na lahat ng mga setting sa lahat ng mga class ang sama-samang i-merge, 
+  ang kumpigurasyon na natukoy sa anak na class ay magagamit. Ito ay 
+  nangangahulugan na kung mayroon kang ilang kumpigurasyon na tinukoy sa isang
+  subclass, ang kumpigurasyon lamang sa subclass ang magagamit.
+- Ang ``Controller::httpCodes()`` ay tinanggal, sa halip ay gamitin ang
+  :php:meth:`Cake\\Network\\Response::httpCodes()`.
+- Ang ``Controller::disableCache()`` ay tinanggal, sa halip ay gamitin ang
+  :php:meth:`Cake\\Network\\Response::disableCache()`.
+- Ang ``Controller::flash()`` ay tinanggal. Ang paraang ito ay bihira lamang
+  ginamit sa tunay na mga aplikasyon at walang nang layunin na pinagsisilbihan.
+- Ang ``Controller::validate()`` at ``Controller::validationErrors()`` ay
+  tinanggal. Sila ay mga tirang mga paraan mula sa 1.x na kapanahunan kung saan
+  ang mga alalahanin ng mga model + mga controller ay malayong mas magkaakibat.
+- Ang ``Controller::loadModel()`` ngayon ay naglo-load ng table na mga object.
+- Ang ``Controller::$scaffold`` na katangian ay tinanggal. Ang dynamic scaffolding
+  ay tinanggal mula sa core ng CakePHP. Isang pinaunlad na scaffolding na plugin,
+  na nakapangalang CRUD, ay maaaring matagpuan dito:
+  https://github.com/FriendsOfCake/crud
+- Ang ``Controller::$ext`` na katangian ay tinanggal. Ngayon kailangan mong palawigin
+  at i-override ang ``View::$_ext`` na katangian kung gusto mong gumamit ng isang 
+  hindi default na view file na ekstensyon.
+- Ang ``Controller::$methods`` na katangian ay tinanggal. Dapat mo na ngayong
+  gamitin ang ``Controller::isAction()`` upang matukoy kung ang pangalan ng 
+  paraan ay isang aksyon o hindi. Ang pagbabagong ito ay ginawa upang payagan
+  ang mas madaling pag-customize ng kung ano at kung ano ang hindi ang binibilang
+  bilang isang aksyon.
+- Ang ``Controller::$Components`` na katangian ay tinanggal at pinalitan ng 
+  ``_components``. Kung kailangan mong mag-load ng mga komponent sa runtime dapat
+  kang gumamit ng ``$this->loadComponent()`` sa iyong controller.
+- Ang lagda ng :php:meth:`Cake\\Controller\\Controller::redirect()` ay binago
+  sa ``Controller::redirect(string|array $url, int $status = null)``. Ang 
+  pangatlong argumento na ``$exit`` ay tinanggal. Ang paraan ay hindi na nagpapadala
+  ng tugon at exit na iskrip, sa halip ito ay nagsasauli ng isang ``Response``
```

Comment:
exit to palabas

Reply:
I used "labasan" for this instead. the word "palabas" could give another meaning.